src/VisualStudio/IntegrationTest/IntegrationTests/Workspace/WorkspacesNetCore.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests.Workspace
+{
+    [Collection(nameof(SharedIntegrationHostFixture))]
+    public class WorkspacesNetCore : AbstractEditorTest
+    {
+        protected override string LanguageName => LanguageNames.CSharp;
+
+        public WorkspacesNetCore(VisualStudioInstanceFactory instanceFactory)
+            : base(instanceFactory, nameof(WorkspacesNetCore), WellKnownProjectTemplates.CSharpNetCoreClassLibrary)
+        {
+            EnableFullSolutionAnalysis();
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Workspace)]
```

Comment:
Aren't these the same tests as the other Workspaces tests? If so, please use xunit's support for inheritance to share the code.